robotd/devices.py

```diff
@@ -243,8 +241,14 @@ class ServoAssembly(Board):
     INPUT = 'hi-z'
 
     @classmethod
+    def included(cls, node):
+        return node['ID_MODEL_ID'] in ['0043', '7523'] and node['ID_VENDOR_ID'] in ['2a03', '1a86']
```

Comment:
This isn't ideal—it would be better to use the `lookup_keys` system to avoid pulling in every tty device—but it'll probably do for now.

Would mildly prefer:

1. Splitting this over multiple lines,
2. Testing for the model/vendor pairs rather than separately.

Approximately:

```python
return node['ID_MODEL_ID'], node['ID_VENDOR_ID']) in [
    ('0043', '2a03'),  # Fake Uno
    ('7523', '1a86'),  # Real Uno
]